examples/gno.land/p/gov/dao/types.gno

```diff
@@ -0,0 +1,43 @@
+package dao
+
+import (
+	"std"
+
+	"gno.land/p/gov/proposal"
+)
+
+type VoteOption string
+
+const (
+	YesVote VoteOption = "YES"
+	NoVote  VoteOption = "NO"
+)
+
+func (v VoteOption) String() string {
+	return string(v)
+}
+
+// Vote encompasses a single GOVDAO vote
+type Vote struct {
```

Comment:
move to vote.gno

Reply:
Moved:

[3a7ba00](https://github.com/gnolang/gno/pull/2581/commits/3a7ba0041c0a8c1fe8375911c77ee2eb9ab951ed)